anthos-baremetal-edge-deployment/scripts/create-primary-gsa.sh

```diff
@@ -66,16 +67,6 @@ gcloud projects add-iam-policy-binding $PROJECT_ID \
     --member="serviceAccount:${GSA_EMAIL}" \
     --role="roles/secretmanager.secretAccessor" --no-user-output-enabled
 
-echo "Adding roles/secretmanager.secretAccessor"
-gcloud projects add-iam-policy-binding $PROJECT_ID \
-    --member="serviceAccount:${GSA_EMAIL}" \
-    --role="roles/gkehub.gatewayAdmin" --no-user-output-enabled
-
-echo "Adding roles/secretmanager.secretAccessor"
-gcloud projects add-iam-policy-binding $PROJECT_ID \
-    --member="serviceAccount:${GSA_EMAIL}" \
-    --role="roles/gkehub.viewer" --no-user-output-enabled
-
```

Comment:
Question: Is the removal of this chunk of `.sh` scripts intentional?

Reply:
omgosh i def removed this by accident great catch 👏  ill add it back in!!